### DISCUSSION TITLE
360 controls with QRControl app

### DISCUSSION THREAD

**Discussion by dwightbowman** [gopro/labs#848] [Feature Requests]
How can I control my MAX 360 settings with the QRControl app or QRcodes in general.  The menus in QRControl app and the web page don't appear to control the settings for 360 mode.

**Comment by dnewman-gpsw** [comment]
It is all there. Res 5K is for 5.6K on MAX,  and 3K 360° clear only has a single MAX usage.

![image](https://github.com/gopro/labs/assets/10856084/82361275-a2ba-47cf-8b7f-03460a7bf7f0)

All Protune settings and triggers should work fine on MAX.  About 80-90% will work. However, no camera supports 100% of QRControl.

**Reply by dwightbowman** [reply]
I missed those settings. (There are alot of them :-) )  Thanks for the informative reply!
